modules/openshift-developer-cli-pushing-the-odo-init-image-to-a-mirror-registry.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/pushing-the-odo-init-image-to-the-restricted-cluster-registry.adoc
+
+[id="pushing-the-odo-init-image-to-a-mirror-registry_{context}"]
+= Pushing the `odo init` image to a mirror registry
+
+Push the `odo init` image to a cluster with a mirror registry as follows:
```

Comment:
Suggest adding, Depending on your operating system, you can push...

Reply:
Thank you. Changed. 

Also changed wording of a phrase about `ODO_BOOTSTRAPPER_IMAGE` throughout the docs.